gladier_xpcs/tools/corr_gpu.py

```diff
@@ -0,0 +1,60 @@
+from gladier import GladierBaseTool, generate_flow_definition
+
+
+def eigen_corr_gpu(**event):
+    import os
+    import subprocess
+    from subprocess import PIPE
+
+    ##minimal data inputs payload
+    proc_dir = event.get('proc_dir') # location of transfered files / results
+    raw_file = event.get('raw_file') # raw data
+    qmap_file = event.get('qmap_file') # name of the qmap file
+    corr_gpu_loc = event.get('corr_gpu_loc') #location of processing script
+
+    if not os.path.exists(proc_dir):
+        raise NameError(f'{proc_dir} \n Proc dir does not exist!')
+
+    os.chdir(proc_dir)
+
+    flags = "--batch_size 256 --verbose" #TODO how to pick batch size?
```

Comment:
Should event['flags'] be added here? `--verbose` seems like a good optional one to come through flow input.

Reply:
+1